src/drc/api/tests/test_enkelvoudiginformatieobject.py

```diff
@@ -306,7 +305,10 @@ def test_filter_zoek_uuids(self):
         eio1 = EnkelvoudigInformatieObjectFactory.create(identificatie="foo")
         eio2 = EnkelvoudigInformatieObjectFactory.create(identificatie="bar")
         eio3 = EnkelvoudigInformatieObjectFactory.create(identificatie="bar2")
-        response = self.client.get(self.list_url, {"uuid": [eio2.uuid, eio1.uuid]})
+        response = self.client.get(self.list_url, {"uuidIn": [eio2.uuid, eio1.uuid]})
+        from pprint import pprint
+
+        pprint(response.json())
```

Comment:
```suggestion
```

Reply:
Voor zaken was de mogelijkheid opm te filteren op een lijst met uuids reden voor een PR: #1728

Wat is hier dan anders?

Reply:
Het gaat hier over query parameters, in tegenstelling tot in de PR van [#1728](https://github.com/VNG-Realisatie/zaken-api/pull/169/files), waar het gaat om request body parameters.

Reply:
Maar had dat dan niet ook opgelost kunnen worden met een query parameter ipv een request body parameter?  Of andersom, had dit dan niet opgelost moeten worden met een request body parameter?

Nu hebben we twee verschillende manieren om hetzelfde te doen, dat geeft misschien verwarring bij de gebruikers?

Reply:
Volgens mij is hier voor destijds gekozen omdat `UUID`'s vrij lange waarden zijn en je hierdoor snel tegen het limit aanliep van het maximaal aantal characters voor een URL als je query parameters gebruikt. In een request body heb je dit euvel niet maar kun je niet per definitie per attribuut meerdere waarden opgeven (tenzij het veld hiervoor is ingericht uiteraard).

Reply:
Dat klinkt als een goede reden. Moeten we dit dan niet ook toepassen op Informatieobjecten en een aparte POST operatie maken (enkelvoudigifnroamtieobject_zoek oid) waaraan een request body parameter meegegeven kan worden?

Het probleem is: Zodra het er een keer in zit kunnen we het niet meer weghalen zonder breaking change.